posthog/api/capture.py

```diff
@@ -89,12 +89,16 @@ def _capture(request, team: Team, event: str, distinct_id: str, properties: Dict
         event=event,
         distinct_id=distinct_id,
         properties=properties,
-        ip=get_ip_address(request),
         team=team,
         **({'timestamp': timestamp} if timestamp else {}),
         **({'elements': elements_list} if elements_list else {})
     )
 
+    ip=get_ip_address(request)
```

Comment:
I want to explore further, if this can be set in array.js.

Reply:
I think that'll make it too easy for it to be spoofed. I think you're current solution makes sense.